Apps/WebClient/src/ClientApp/test/stubs/store/vaccinationStatusStub.ts

```diff
@@ -105,7 +109,7 @@ const vaccinationStatusMutations: VaccinationStatusMutations = {
     setVaccinationStatus: voidMethod,
     vaccinationStatusError: voidMethod,
     setPdfRequested: voidMethod,
-    pdfError: voidMethod,
+    setPublicVaccineRecordError: voidMethod,
```

Comment:
Nit: suggest moving this below with the other `setPublicVaccineRecord*` mutations for organization.

Reply:
Also, the `vaccinationStatusActions` above need to be updated to include the new actions.

Reply:
@MikeLyttle  the 1st comment I applied the change but I am bit confused on the 2nd comment?

Reply:
Further up in the file, we mock out the actions for this store module in `const vaccinationStatusActions`, but we forgot to add in the new actions related to the public vaccine record.